logbus/logstreamer/helheim_test.go

```diff
@@ -0,0 +1,144 @@
+// Code generated by git.sr.ht/~nelsam/hel/v4. DO NOT EDIT.
```

Comment:
There are just mocks in here...Should we add `mock` to the test name?

Reply:
We can. So far, we've just [used the default name](https://github.com/earthly/earthly/blob/main/ast/helheim_test.go), but `hel` does have a flag to override that.

`helheim_mocks_test.go`?

Reply:
Yeah, I like `helheim_mocks_test.go`.

I don't mind helheim being in the name - I like the word `mock(s)` in the name for the same reason. 
While playful, and I like the nod to Norse mythology, it may not be as obvious as `mock(s)` to one unfamiliar with it.